kvmagent/kvmagent/test/utils/nfs_plugin_utils.py

```
@@ -40,7 +40,7 @@ def create_root_volume_from_template(templatePathInCache, timeout, virtualSize,
 
 @misc.return_jsonobject()
 def migrate_bits(srcFolderPath, dstFolderPath, independentPath=False, filtPaths=[], isMounted=True, volumeInstallPath=None, options=None,
-                 url=None, mountPath=None, kvmHostAddons={}):
+                 url=None, mountPath=None, kvmHostAddons={"qcow2Options":""}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)